FinniversKit/Sources/Components/SideScrollableView/SideScrollableView.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  Copyright © 2022 FINN.no AS. All rights reserved.
+//
+
+import Combine
+import UIKit
+
+public struct SideScrollableItemModel {
+    
+    let items: [String]
+    
+    public init(items: [String]) {
+        self.items = items
+    }
+}
```

Comment:
I think this could be pulled out into another file, just to keep models and the view separated :)